app/assets/stylesheets/content/_editable_toolbar.sass

```diff
@@ -13,6 +13,11 @@
     .button
       margin: 0
 
+.title-container.-small
+  .editable-toolbar-title--fixed
+    font-size: 18px !important
+    height: 22px
```

Comment:
The numbers do not align with the hover icons. Either change them here, or change the position of the hover icons. Please remove the styles from [here](https://github.com/opf/openproject/blob/dev/frontend/src/app/modules/common/editable-toolbar-title/editable-toolbar-title.sass#L9), too. Also I do not think that the `!important` is necessary.

<img width="309" alt="Bildschirmfoto 2019-05-28 um 15 02 30" src="https://user-images.githubusercontent.com/7457313/58481136-0dcca300-815c-11e9-9c9e-1c5955b43ed9.png">

Reply:
I'm not going to provide a pixel perfect positioning for the icons as long as they are bound to change. Will try to pragmatically remove styles that appear unnecessary